openassessment/xblock/test/test_studio.py

```diff
@@ -254,8 +274,8 @@ def test_render_editor_assessment_order(self, xblock):
         self._assert_rendered_editor_order(xblock, [
             'example-based-assessment',
             'student-training',
-            'self-assessment',
             'peer-assessment',
+            'self-assessment',
```

Comment:
New behavior - will force the `editor_assessments_order` to match the assessments defined in the problem. In this case, `data/basic_scenario.xml` defines peer before self.